vernac/vernacinterp.mli

```diff
@@ -27,3 +27,5 @@ val test_mode : bool ref
 (** Default proof mode set by `start_proof` *)
 val get_default_proof_mode : unit -> Pvernac.proof_mode
 val proof_mode_opt_name : string list
+
+val get_program_state : unit -> Declare.OblState.t NeList.t
```

Comment:
This is in particular the bit that bothers me here.

Reply:
If I understand you correctly. Here, now plugins and other users of the API can access this without having to declare it in the type.

Reply:
We can expose just something to `check_solved_obligations` if you prefer.
Plugins could already link stm so there was no capability change imo.